drivers/sensor/realtek/rts5912/rts5912.c

```diff
@@ -50,7 +50,7 @@ int tach_rts5912_sample_fetch(const struct device *dev, enum sensor_channel chan
 
 	tach->status = TACHO_STS_CNTRDY;
 
-	if (WAIT_FOR(tach->status & TACHO_STS_CNTRDY, PIN_STUCK_TIMEOUT, k_msleep(1))) {
+	if (WAIT_FOR(tach->status & TACHO_STS_CNTRDY, PIN_STUCK_TIMEOUT, k_msleep(2))) {
```

Comment:
> we found that the tachometer is too often to check the fan speed. It makes the system is hard to enter sleep mode or wake up too soon, therefore we decrease the frequency of checking the fan speed value.

Ok maybe add a coment line about it here? Even better, add a define for the number and comment on the define itself.